### ISSUE TITLE
Crash While Exiting While Docked

### ISSUE THREAD

**Issue by ToxiClay** [16BitWonder/NX-Update-Checker#33]
# Bug Report

## Switch Environment
System Version: **15.0.0**

Atmosphere Version: **1.4.0 (loaded on sysNAND using fss0)**

Additional sysmodules being loaded: 

1. sys-ftpd-light
2. SysDVR
3. sys-clk
4. nx-ovlloader

Blanking Prodinfo/Incognito: **No**

## Describe the Bug
While running NX-Update-Checker 1.5.4 in Applet mode **while the system is docked**, pressing `+` to exit the app will cause Atmosphere to crash. Running NX-Update Checker via title override causes the title to crash "because an error occurred."

**Interestingly,** running NX-Update-Checker with `log.flag` present avoids the crash.

I have reproduced the above behavior consistently.

## To Reproduce
Please outline the steps below to reproduce this issue:

1. Have the Switch docked.
2. Load Atmosphere onto the system through whichever means desired.
3. Access the HB Menu using the Album applet.
5. Launch NX-Update-Checker and let it run.
6. Press `+` to quit.

## Produced Logs & Lists

[01667342038_010000000000100d.log](https://github.com/16BitWonder/NX-Update-Checker/files/9914450/01667342038_010000000000100d.log)
[NXUC.log](https://github.com/16BitWonder/NX-Update-Checker/files/9914451/NXUC.log)
[versions.txt](https://github.com/16BitWonder/NX-Update-Checker/files/9914452/versions.txt)

**Comment by Konzolista** [comment]
**Please remove this error, the same is happening to me...**

**Comment by proletariado** [comment]
Yup, can confirm. Works fine in handheld mode, crashes in the dock.
